docs/circuit_cutting/tutorials/gate_cutting_to_reduce_circuit_depth.ipynb

```diff
@@ -0,0 +1,423 @@
+    "\n",
+    "To simulate the expectation value of the full-sized circuit, many subexperiments are generated from the decomposed gates' joint quasiprobability distribution and then executed on one or more backends. The number of samples taken from the distribution is controlled by `num_samples`. If the smallest sample probability in the joint quasi-probability distribution is above 1 / `num_samples`, exact weights will be used.  Otherwise, the weights will be sampled, resulting in at most `num_samples` unique samples.  In the future, this function will be changed to return a mixture of exact and sampled weights, so that the largest weights will be calculated exactly, while weights in the tail of the distribution will be sampled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "80dd2a66",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "\n",
+    "# Keep in mind, Terra Sampler does not support mid-circuit measurements at all,\n",
+    "# and Aer Sampler does not support mid-circuit measurements when shots==None.\n",
+    "sampler = Sampler(run_options={\"shots\": 2**12})\n",
+    "\n",
+    "quasi_dists, coefficients = execute_experiments(\n",
+    "    circuits=qpd_circuit,\n",
+    "    subobservables=observables,\n",
+    "    num_samples=1500,\n",
+    "    samplers=sampler,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76b89a2f",
+   "metadata": {},
+   "source": [
+    "`execute_experiments` returns:\n",
+    " - A 3D list of length-2 tuples containing a quasiprobability distribution and QPD bit information for each unique subexperiment.\n",
+    " - The coefficients for each subexperiment are also returned. If a sufficient number of samples is taken, the returned coefficients will be calculated exactly; otherwise, the sampled weight will be used."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "319c25d7",
+   "metadata": {},
+   "source": [
+    "### Reconstruct the simulated expectation values\n",
+    "\n",
+    "`reconstruct_expectation_values` expects `quasi_dists` and `coefficients` in the same format as returned from `execute_experiments`. `quasi_dists` is a 3D list of shape (num_unique_samples, num_partitions, num_commuting_observ_groups), and `coefficients` is a list with length equal to the number of unique samples. `subobservables` is the dictionary mapping qubit partition label to the associated subobservable(s). The decomposition of this circuit did not result in separation into subcircuits, so there will be only one partition represented in these outputs."
```

Comment:
```suggestion
    "`reconstruct_expectation_values` expects `quasi_dists` and `coefficients` in the same format as returned from `execute_experiments`. `quasi_dists` is a 3D list of shape (`num_unique_samples`, `num_partitions`, `num_commuting_observ_groups`), and `coefficients` is a list with length equal to the number of unique samples. `subobservables` is the dictionary mapping qubit partition label to the associated subobservable(s). The decomposition of this circuit did not result in separation into subcircuits, so there will be only one partition represented in these outputs."
```

Reply:
@garrison I took this out at some point because the highlights get a bit dense in this paragraph, but I do agree these should be highlighted. That was my initial thought too

Reply:
ha, I was just looking at it thinking the same thing before I saw this.  We can re-evaluate how this is presented in the future.